python/pylibvw.cc

```diff
@@ -136,15 +137,15 @@ size_t my_get_label_type(vw*all)
 
 size_t my_get_prediction_type(vw_ptr all)
 { switch (all->l->pred_type)
-  { case prediction_type::scalar:          return pSCALAR;
-    case prediction_type::scalars:         return pSCALARS;
-    case prediction_type::action_scores:   return pACTION_SCORES;
-    case prediction_type::action_probs:    return pACTION_PROBS;
-    case prediction_type::multiclass:      return pMULTICLASS;
-    case prediction_type::multilabels:     return pMULTILABELS;
-    case prediction_type::prob:            return pPROB;
-    case prediction_type::multiclassprobs: return pMULTICLASSPROBS;
-    case prediction_type::decision_probs:  return pDECISION_SCORES;
+  { case prediction_type_t::scalar:          return pSCALAR;
```

Comment:
We can/should expose these through enums in the bindings (boost_python and pybind both have the concept of python enums). Since thats an interface breaking change though, probably add it as a secondary method and deprecate this one

Reply:
I agree, that would be a better way of exposing